kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/FileNames.java

```diff
@@ -111,6 +131,20 @@ public static long deltaVersion(String path) {
     return Long.parseLong(name.split("\\.")[0]);
   }
 
+  /** Returns the start and end versions for the given compaction path. */
+  public static Tuple2<Long, Long> logCompactionVersions(Path path) {
+    final String[] split = path.getName().split("\\.");
+    return new Tuple2<>(Long.parseLong(split[0]), Long.parseLong(split[1]));
+  }
+
+  public static Tuple2<Long, Long> logCompactionVersions(String path) {
+    return logCompactionVersions(new Path(path));
+  }
+
+  // public static boolean deltaIsCoveredBy(String deltaPath, List<String> compactionFiles) {
```

Comment:
should we clean this up?

Reply:
Yep that slipped through a refactor cleanup. Thanks, nice catch